### DISCUSSION TITLE
PREEMPT_RT patch for Polarfire SoC Linux

### DISCUSSION THREAD

**Discussion by aurang-microchip** [polarfire-soc/polarfire-soc-documentation#280] [Linux build systems for PolarFire SoC] [answered]
Hi,
I have a customer who tried to use PREEMPT_RT patch using our yocto Linux distribution. He wants to use realtime capable Linux.
It didn't work and resulted in a kernel panic. The log is available and can be shared.
Do we support/recommend using this patch or do we guide the Customer to other real-time options like using our RTOS offerings etc.
Thanks!

Best Regards,
Aurang

**Comment by hughbreslin** [comment, marked as answer]
Hey @aurang-microchip we haven't looked at any of the RT patches yet really. The best advice would be to look at the RTOS / AMP offerings to implement real time :)
